_bibliography/citations-eu.bib

```diff
@@ -6522,3 +6522,94 @@ @article{zirngibl_triose_2023
  volume = {4},
  year = {2023}
 }
+
+@article{simon2022novel,
+ abstract = {Macrophages (MΦ), known for immunological roles, such as phagocytosis and antigen presentation, have been found to electrotonically couple to cardiomyocytes (CM) of the atrioventricular node via Cx43, affecting cardiac conduction in isolated mouse hearts. Here, we characterize passive and active electrophysiological properties of murine cardiac resident MΦ, and model their potential electrophysiological relevance for CM.We combined classic electrophysiological approaches with 3D florescence imaging, RNA-sequencing, pharmacological interventions, and computer simulations. We used Cx3cr1eYFP/+ mice wherein cardiac MΦ are fluorescently labelled. FACS-purified fluorescent MΦ from mouse hearts were studied by whole-cell patch-clamp. MΦ electrophysiological properties include: membrane resistance 2.2±0.1 GΩ (all data mean±SEM), capacitance 18.3±0.1 pF, resting membrane potential −39.6±0.3 mV, and several voltage-activated, outward or inwardly rectifying potassium currents. Using ion channel blockers (barium, TEA, 4-AP, margatoxin, XEN-D0103, and DIDS), flow cytometry, immuno-staining, and RNA-sequencing, we identified Kv1.3, Kv1.5, and Kir2.1 as channels contributing to observed ion currents. MΦ displayed four patterns for outward and two for inward-rectifier potassium currents. Additionally, MΦ showed surface expression of Cx43, a prerequisite for homo- and/or heterotypic electrotonic coupling. Experimental results fed into development of an original computational model to describe cardiac MΦ electrophysiology. Computer simulations to quantitatively assess plausible effects of MΦ on electrotonically coupled CM showed that MΦ can depolarize resting CM, shorten early and prolong late action potential duration, with effects depending on coupling strength and individual MΦ electrophysiological properties, in particular resting membrane potential and presence/absence of Kir2.1.Our results provide a first electrophysiological characterization of cardiac resident MΦ, and a computational model to quantitatively explore their relevance in the heterocellular heart. Future work will be focussed at distinguishing electrophysiological effects of MΦ–CM coupling on both cell types during steady-state and in patho-physiological remodelling, when immune cells change their phenotype, proliferate, and/or invade from external sources.},
```

Comment:
this one is already included.

Reply:
ok, I will remove it. Should I also reorder them based on the string?